packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -822,19 +822,22 @@ Iterable<File> findApkFiles(GradleProject project, AndroidBuildInfo androidBuild
 
 File _findBundleFile(GradleProject project, BuildInfo buildInfo) {
```

Comment:
Can you make this `@visibleForTesting` and add unit tests? I see gradle_test.dart already has tests for e.g. `findApkFiles` above.

Reply:
I added a few unit tests :)